examples/smart-contract-upgrade/contract-version1/src/lib.rs

```diff
@@ -47,7 +47,7 @@ struct UpgradeParams {
 
 /// Smart contract errors.
 #[derive(PartialEq, Eq, Reject, Serial, SchemaType)]
-enum CostumContractError {
+enum CustomContractError {
```

Comment:
Is the `smart-contract-upgade` contract related to the credential storage contract?

Reply:
It is just a leftover comment from my last MR:
https://github.com/Concordium/concordium-rust-smart-contracts/pull/262#discussion_r1182565655